keyboards/rart/rart45/info.json

```
@@ -0,0 +1,70 @@
+{
+  "keyboard_name": "Rartlice",
+  "url": "",
+  "maintainer": "Alabahuy",
+  "width": 13,
+  "height": 5,
+  "layouts": {
+    "LAYOUT_all": {
+      "layout": [
+        { "x": 0, "y": 0 },
+        { "x": 1, "y": 0 },
+        { "x": 2, "y": 0 },
+        { "x": 3, "y": 0 },
+        { "x": 4, "y": 0 },
+        { "x": 5, "y": 0 },
+        { "x": 7, "y": 0 },
+        { "x": 8, "y": 0 },
+        { "x": 9, "y": 0 },
+        { "x": 10, "y": 0 },
+        { "x": 11, "y": 0 },
+        { "x": 12, "y": 0 },
+
+        {"x": 0, "y": 1 },
+        {"x": 1, "y": 1 },
+        {"x": 2, "y": 1 },
+        {"x": 3, "y": 1 },
+        {"x": 4, "y": 1 },
+        {"x": 5, "y": 1 },
+        {"x": 7, "y": 1 },
+        {"x": 8, "y": 1 },
+        {"x": 9, "y": 1 },
+        {"x": 10, "y": 1 },
+        {"x": 11, "y": 1 },
+        {"x": 12, "y": 1 },
+
+        {"x": 0, "y": 2 },
+        {"x": 1, "y": 2 },
+        {"x": 2, "y": 2 },
+        {"x": 3, "y": 2 },
+        {"x": 4, "y": 2 },
+        {"x": 5, "y": 2 },
+        {"x": 7, "y": 2 },
+        {"x": 8, "y": 2 },
+        {"x": 9, "y": 2 },
+        {"x": 10, "y": 2 },
+        {"x": 11, "y": 2 },
+        {"x": 12, "y": 2 },
+
+        {"x": 0, "y": 3 },
+        {"x": 1, "y": 3 },
+        {"x": 2, "y": 3 },
+        {"x": 3, "y": 3 },
+        {"x": 4, "y": 3 },
+        {"x": 5, "y": 3 },
+        {"x": 7, "y": 3 },
+        {"x": 8, "y": 3 },
+        {"x": 9, "y": 3 },
+        {"x": 10, "y": 3 },
+        {"x": 11, "y": 3 },
+        {"x": 12, "y": 3 },
+
+        {"x": 1.75, "y": 4, "w":1.25},
+        {"x": 3, "y": 4, "w":1.25 },
+        {"x": 4.25, "y": 4, "w":2.25 },
+        {"x": 6.5, "y": 4, "w":1.25 },
+        {"x": 7.75, "y": 4, "w":1.25 }
```

Centering these keys.

```suggestion
        {"x": 2.875, "y": 4, "w":1.25},
        {"x": 4.125, "y": 4, "w":1.25 },
        {"x": 5.375, "y": 4, "w":2.25 },
        {"x": 7.625, "y": 4, "w":1.25 },
        {"x": 8.875, "y": 4, "w":1.25 }
```

---

thanks @noroadsleft , done for fix it